### ISSUE TITLE
:boom:  !Execution Failed

### ISSUE THREAD

**Issue by d3v1l0n** [Snigdha-OS/snigdhaos-kernel-manager#1]
```python
$ /bin/python /home/whoami/Documents/GitHub/snigdhaos-kernel-manager/snigdhaos-kernel-manager.py
MESA: error: ZINK: vkCreateInstance failed (VK_ERROR_INCOMPATIBLE_DRIVER)
libEGL warning: egl: failed to create dri2 screen
2024-06-03 03:34:53:ERROR > Exception in make log directory(): [Errno 13] Permission denied: '/var/log/snigdhaos-kernel-manager'
Traceback (most recent call last):
  File "/home/whoami/Documents/GitHub/snigdhaos-kernel-manager/snigdhaos-kernel-manager.py", line 4, in <module>
    import functions as fn
  File "/home/whoami/Documents/GitHub/snigdhaos-kernel-manager/functions.py", line 261, in <module>
    tfh = TimedRotatingFileHandler(event_log_file, encoding="utf-8", delay=False, when="W4")
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/logging/handlers.py", line 214, in __init__
    BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
  File "/usr/lib/python3.12/logging/handlers.py", line 58, in __init__
    logging.FileHandler.__init__(self, filename, mode=mode,
  File "/usr/lib/python3.12/logging/__init__.py", line 1231, in __init__
    StreamHandler.__init__(self, self._open())
                                 ^^^^^^^^^^^^
  File "/usr/lib/python3.12/logging/__init__.py", line 1263, in _open
    return open_func(self.baseFilename, self.mode,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/var/log/snigdhaos-kernel-manager/event.log'
```

**Comment by d3v1l0n** [comment]
@iconized ki bhai? din kal valo to?

**Comment by eshaanized** [comment]
need time to fix #1

**Comment by eshaanized** [comment]
> ```python
> $ /bin/python /home/whoami/Documents/GitHub/snigdhaos-kernel-manager/snigdhaos-kernel-manager.py
> MESA: error: ZINK: vkCreateInstance failed (VK_ERROR_INCOMPATIBLE_DRIVER)
> libEGL warning: egl: failed to create dri2 screen
> 2024-06-03 03:34:53:ERROR > Exception in make log directory(): [Errno 13] Permission denied: '/var/log/snigdhaos-kernel-manager'
> Traceback (most recent call last):
>   File "/home/whoami/Documents/GitHub/snigdhaos-kernel-manager/snigdhaos-kernel-manager.py", line 4, in <module>
>     import functions as fn
>   File "/home/whoami/Documents/GitHub/snigdhaos-kernel-manager/functions.py", line 261, in <module>
>     tfh = TimedRotatingFileHandler(event_log_file, encoding="utf-8", delay=False, when="W4")
>           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/lib/python3.12/logging/handlers.py", line 214, in __init__
>     BaseRotatingHandler.__init__(self, filename, 'a', encoding=encoding,
>   File "/usr/lib/python3.12/logging/handlers.py", line 58, in __init__
>     logging.FileHandler.__init__(self, filename, mode=mode,
>   File "/usr/lib/python3.12/logging/__init__.py", line 1231, in __init__
>     StreamHandler.__init__(self, self._open())
>                                  ^^^^^^^^^^^^
>   File "/usr/lib/python3.12/logging/__init__.py", line 1263, in _open
>     return open_func(self.baseFilename, self.mode,
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> FileNotFoundError: [Errno 2] No such file or directory: '/var/log/snigdhaos-kernel-manager/event.log'
> ```

`sudo` permission is necessary, try to run as sudo, if failed, paste the error log here like before!
